less/styles.less

```diff
@@ -173,9 +173,6 @@
     & {
       background-color: #000;
     }
-    h2, p {
-      color: @white;
-    }
```

Comment:
I am removing this because the text color is white on JakartaEE on light background, cannot see clearly